### PR TITLE
Fixes #103 - Updated links to public static assets.

### LINKED ISSUE #103: Host static assets on a CDN
Hotlinking to Github static assets hosting doesn't work well with browsers having strong mimetype checks. Also that's probably unfair to Github, so we have to find a proper solution.

### PR DESCRIPTION
**Pull request by n1k0**
Refs #103. r=? @Natim

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -14,7 +14,14 @@ $ npm install kinto --save-dev
...
-Dev and production ready assets are available in the [`dist` folder of the repository](https://github.com/Kinto/kinto.js/tree/master/dist). You can either download an archive of the code and use these file in your own project, or link to static assets hosted on github:
+Dev and production ready assets are available in the [`dist` folder of the repository](https://github.com/Kinto/kinto.js/tree/master/dist). You can either download an archive of the code and use these files in your own project, or link to static assets served through [rawgit](http://rawgit.com/):
 
-- Dev version: [latest master](https://raw.githubusercontent.com/Kinto/kinto.js/master/dist/kinto.dev.js)
-- Production version: [latest master](https://raw.githubusercontent.com/Kinto/kinto.js/master/dist/kinto.min.js)
+### Latest master
+
+- Dev version, including source maps: [kinto.dev.js](https://rawgit.com/Kinto/kinto.js/master/dist/kinto.dev.js)
+- Production version, minified, no source maps: [kinto.dev.js](https://rawgit.com/Kinto/kinto.js/master/dist/kinto.min.js)
+
```
Comment (Natim) [inline docs/installation.md]: We should add a note telling that the master version should not be used in production or linked in any code but may be used for testing. For production people should make sure to use the latest tag.

Comment (Natim) [inline docs/installation.md]: Maybe we should not inform people that they can use the master version and just point the CDN for tagged version?

Comment (n1k0) [inline docs/installation.md]: My plan was to host different version of the demo, adding one always using latest master.

Comment (Natim) [inline docs/installation.md]: Or you can use it for your demo but not document it :)

**Comment by n1k0** [comment]
As tagged versions are for now not much more stable than master, I'm gonna land this as an issue-tracker-cleanup process :)

**Pull request merged**